modules/cnf-scheduling-workload-onto-worker-with-real-time-capabilities.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc
+
+[id="cnf-scheduling-workload-onto-worker-with-real-time-capabilities_{context}"]
+= Scheduling a low latency workload onto a worker with real-time capabilities
+
+You can schedule low latency workloads onto a worker node where a performance profile that configures real-time capabilities is applied.
+
+[NOTE]
+====
+To schedule the workload on specific nodes, use label selectors in the `Pod` custom resource (CR).
+The label selectors must match the nodes that are attached to the machine config pool that was configured for low latency by the Node Tuning Operator.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+* You have applied a performance profile in the cluster that tunes worker nodes for low latency workloads.
+
+.Procedure
+
+. Create a `Pod` CR for the low latency workload and apply it in the cluster, for example:
++
+.Example `Pod` spec configured to use real-time processing
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dynamic-low-latency-pod
+  annotations:
+    cpu-quota.crio.io: "disable" <1>
+    cpu-load-balancing.crio.io: "disable" <2>
+    irq-load-balancing.crio.io: "disable" <3>
+spec:
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+  containers:
+  - name: dynamic-low-latency-pod
+    image: "registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version}"
+    command: ["sleep", "10h"]
+    resources:
+      requests:
+        cpu: 2
+        memory: "200M"
+      limits:
+        cpu: 2
+        memory: "200M"
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop: [ALL]
+  nodeSelector:
+    node-role.kubernetes.io/worker-cnf: "" <4>
+  runtimeClassName: performance-dynamic-low-latency-profile <5>
+# ...
+----
+<1> Disables the CPU completely fair scheduler (CFS) quota at the pod run time
+<2> Disables CPU load balancing
+<3> Opts the pod out of interrupt handling on the node
```

Comment:
Periods
<1> Disables the CPU completely fair scheduler (CFS) quota at the pod run time.
<2> Disables CPU load balancing.
<3> Opts the pod out of interrupt handling on the node.

Reply:
https://www.ibm.com/docs/en/ibm-style?topic=format-lists#punctuation-in-lists

Don't use periods when your list contains sentence fragments.

Reply:
Actually no you're right. Full stops it is :sweat_smile: